integration/kubernetes/runtimeclass_workloads/busybox-pod.yaml

```diff
@@ -14,13 +14,19 @@ spec:
   containers:
   - name: first-test-container
     image: busybox
+    env:
+    - name: CONTAINER_NAME
+      value: "first-test-container"
     command:
         - sleep
-        - "30"
+        - "60"
```

Comment:
Why do we need these changes?

Reply:
I'm not sure, but be afraid of if the container will exit during the test are running. In that case kubectl exec maybe get "unable to upgrade connection: container not found" error